KEEP/services/hiawatha

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if ! grep -q '^www-data:' /etc/passwd ; then
+	busybox addgroup -S www-data
+	busybox adduser -h "$SRVDIR" -s /bin/nologin -S -D -H -G www-data www-data
+	if [ ! -d "$SRVDIR" ] ; then
```

Comment:
Fixed - Hiawatha brings `/var/www/index.html` - but you're right, it's safer to create the folder in case someone deleted it there and it isn't more code anyway.